### PR TITLE
Combined three stores into one with Whois struct as value

### LINKED ISSUE #1: Proposal: Collapsing the 3 stores
Currently there are three stores for the nameservice module. I think we should collapse them into one. This would have the following benefits:

- Mirror current best practices in existing modules
- Add a `types.go` file to the module for the struct that would contain `{value, price, owner}`, which would help readers understand its use. They will likely have to have one if they ever build their own module. 
- Unmarshalling overhead is ok since we always use at least 2/3 fields of the struct in functions.

### PR DESCRIPTION
**Pull request by sunnya97**
closes #1

### CODE REVIEW

File: x/nameservice/types.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package nameservice
```
Comment (jackzampolin) [inline x/nameservice/types.go]: Like this whole file.

**Comment by jackzampolin** [comment]
This looks great @sunnya97 we need to update the text of the tutorial for this change and that needs to be in this PR.

**Comment by okwme** [comment]
updated the markdown files on sunny's branch. If he merges those this PR might be ok to merge too?
https://github.com/sunnya97/sdk-application-tutorial/pull/1

**Comment by jackzampolin** [comment]
Lets go ahead and merge when ready @okwme

**Comment by sunnya97** [comment]
Replaced by #43

**Pull request closed without merging**